librz/il/definitions/effect.c

```diff
@@ -0,0 +1,165 @@
+// SPDX-FileCopyrightText: 2021 heersin <teablearcher@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_il/definitions/effect.h>
+
+RZ_API RzILDataEffect effect_new_data(void) {
```

Comment:
Add Doxygen to these functions. If you want to keep them as `RZ_API` it's better to make proper names like `rz_il_effect_new_data()`, etc